shared/triage_metadata.go

```diff
@@ -265,14 +257,14 @@ func generateRandomInt() string {
 }
 
 func (tm triageMetadata) Triage(metadata MetadataResults) (string, error) {
-	filesMap, err := GetMetadataByteMap(tm.logger, tm.fetcher)
+	sha, filesMap, err := GetMetadataByteMap(tm.logger, tm.fetcher)
```

Comment:
We talked about force refreshing the cache before creating the triage PR to minimize the chance of race conditions. Do you still plan to do that?

Reply:
See next [commit](https://github.com/web-platform-tests/wpt.fyi/pull/1927/commits/aeb0f22529d93f6553b8380db765b20a8d96d940)